make node the default js runtime because it is the only one packaged

--- a/yt-dlp.1
+++ b/yt-dlp.1
@@ -124,10 +124,10 @@
 This option can be used multiple times to enable multiple runtimes.
 Supported runtimes are (in order of priority, from highest to lowest):
 deno, node, quickjs, bun.
-Only \[dq]deno\[dq] is enabled by default.
+Only \[dq]node\[dq] is enabled by default.
 The highest priority runtime that is both enabled and available will be
 used.
-In order to use a lower priority runtime when \[dq]deno\[dq] is
+In order to use a lower priority runtime when \[dq]node\[dq] is
 available, --no-js- runtimes needs to be passed before enabling other
 runtimes
 .TP
--- a/yt_dlp/YoutubeDL.py
+++ b/yt_dlp/YoutubeDL.py
@@ -543,7 +543,7 @@
     js_runtimes:       A dictionary of JavaScript runtime keys (in lower case) to enable
                        and a dictionary of additional configuration for the runtime.
                        Currently supported runtimes are 'deno', 'node', 'bun', and 'quickjs'.
-                       If None, the default runtime of "deno" will be enabled.
+                       If None, the default runtime of "node" will be enabled.
                        The runtime configuration dictionary can have the following keys:
                         - path: Path to the executable (optional)
                        E.g. {'deno': {'path': '/path/to/deno'}
@@ -736,7 +736,7 @@
                     raise
 
         # Note: this must be after plugins are loaded
-        self.params['js_runtimes'] = self.params.get('js_runtimes', {'deno': {}})
+        self.params['js_runtimes'] = self.params.get('js_runtimes', {'node': {}})
         self._clean_js_runtimes(self.params['js_runtimes'])
 
         self.params['remote_components'] = set(self.params.get('remote_components', ()))
--- a/yt_dlp/extractor/youtube/_video.py
+++ b/yt_dlp/extractor/youtube/_video.py
@@ -2993,7 +2993,7 @@
 
         if not (requested_clients or excluded_clients) and default_clients == self._DEFAULT_JSLESS_CLIENTS:
             self.report_warning(
-                f'No supported JavaScript runtime could be found. Only deno is enabled by default; '
+                f'No supported JavaScript runtime could be found. Only node is enabled by default; '
                 f'to use another runtime add  --js-runtimes RUNTIME[:PATH]  to your command/config. '
                 f'YouTube extraction without a JS runtime has been deprecated, and some formats may be missing. '
                 f'See  {_EJS_WIKI_URL}  for details on installing one', only_once=True)
--- a/yt_dlp/extractor/youtube/jsc/_builtin/node.py
+++ b/yt_dlp/extractor/youtube/jsc/_builtin/node.py
@@ -67,4 +67,4 @@
 
 @register_preference(NodeJCP)
 def preference(provider: JsChallengeProvider, requests: list[JsChallengeRequest]) -> int:
-    return 900
+    return 1900
--- a/yt_dlp/options.py
+++ b/yt_dlp/options.py
@@ -464,14 +464,14 @@
         callback=_list_from_options_callback,
         type='str',
         callback_kwargs={'delim': None},
-        default=['deno'],
+        default=['node'],
         help=(
             'Additional JavaScript runtime to enable, with an optional location for the runtime '
             '(either the path to the binary or its containing directory). '
             'This option can be used multiple times to enable multiple runtimes. '
             'Supported runtimes are (in order of priority, from highest to lowest): deno, node, quickjs, bun. '
-            'Only "deno" is enabled by default. The highest priority runtime that is both enabled and '
-            'available will be used. In order to use a lower priority runtime when "deno" is available, '
+            'Only "node" is enabled by default. The highest priority runtime that is both enabled and '
+            'available will be used. In order to use a lower priority runtime when "node" is available, '
             '--no-js-runtimes needs to be passed before enabling other runtimes'))
     general.add_option(
         '--no-js-runtimes',
